show_client/interface_transceiver_cli.go

```diff
@@ -137,3 +137,71 @@ func getInterfaceTransceiverLpMode(args sdc.CmdArgs, options sdc.OptionMap) ([]b
 
 	return json.Marshal(entries)
 }
+
+func querySfpPM(intf string) map[string]string {
+	return map[string]string{
+		"name":   intf,
+		"status": "Transceiver performance monitoring not applicable",
+	}
+	// TODO: Implement the logic after we find a device that has transceiver performance monitoring enabled
```

Comment:
Why not implement the logic in this PR? Although the test results are identical, we can verify the code logic.